### DISCUSSION TITLE
Is it possible to just backups specific apps?

### DISCUSSION THREAD

**Discussion by DrPepperBianco** [mrrfv/open-android-backup#127] [General]
Instead of a backup of the whole system, is it possible to instead just create a backup of a specific app?

**Comment by LandWarderer2772** [comment]
i dont think you can right now but
you can use [adb](https://developer.android.com/tools/releases/platform-tools)
read about it here
https://gist.github.com/AnatomicJC/e773dd55ae60ab0b2d6dd2351eb977c1
and
https://developer.android.com/tools/adb#copyfiles
